Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2681,8 +2681,9 @@ DiagnosticLoggingEnabled:
 
 DigitalCredentialsEnabled:
   type: bool
-  status: testable
+  status: preview
   category: dom
+  condition: ENABLE(WEB_AUTHN) && HAVE(DIGITAL_CREDENTIALS_UI)
```

Comment:
You'll need to guard places where you read `digitalCredentialsEnabled()` with the same condition.

Reply:
I removed the condition for now.